notfound/extension.py

```diff
@@ -203,3 +204,9 @@ def setup(app):
     if sphinx.version_info >= (1, 8):
         from sphinx.builders.html import setup_js_tag_helper
         app.connect('html-page-context', setup_js_tag_helper)
+
+    return {
+        'version': str(version),
```

Comment:
version is already a string

Reply:
:man_facepalming:

I removed the cast.